### PR TITLE
Tested demos (with blending and changed animate API)

### PR DESCRIPTION
**Pull request by spodila1**
Modified the following demos with respect to PR #116.
1. sxr-avatar
2. sxr-aravatar
3. sxr-arpet
4. sxr-controls
5. sxr-meshanimation
6. sxr-javascript
7. sxr-keyboard
8. sxr-rendertotexture
9. sxr-solarsystem
10. sxr-multilight

SXR DCO signed off by: Sahithi Podila sah.pod@partner.samsung.com

### CODE REVIEW

File: common.gradle [hunk 1]
```diff
@@ -189,17 +189,12 @@ dependencies {
...
 
     arImplementation 'com.google.protobuf:protobuf-lite:3.0.0'
-	if (file("../../../SXR").exists()) {
-        if (project.hasProperty("useLocalDependencies") && project.useLocalDependencies) {
-            if (findProject(':sxr_adapter')) {
-                arImplementation project(':sxr_adapter')
-            } else {
-                arImplementation(':sxr_adapter-release@aar')
-            }
+    if (project.hasProperty("useLocalDependencies") && project.useLocalDependencies) {
```
Comment (liaxim) [inline common.gradle]: What is the purpose of this change?

File: sxr-arpet/app/build.gradle [hunk 2]
```diff
@@ -42,7 +42,7 @@ android {
         buildConfigField "String", "GOOGLE_CLOUD_ANCHOR_KEY_NAME", "\"$GOOGLE_CLOUD_ANCHOR_KEY_NAME\""
         manifestPlaceholders = [cloudAnchorApiKeyName : GOOGLE_CLOUD_ANCHOR_KEY_NAME,
                                 cloudAnchorApiKeyValue: getPropertyOrDefault("cloudAnchorApiKey", "")]
-        minSdkVersion 26
+        minSdkVersion 21
```
Comment (liaxim) [inline sxr-arpet/app/build.gradle]: Why 21 and not 26?

File: sxr-multilight/app/src/main/java/com/samsungxr/multilight/MultiLightMain.java [hunk 3]
```diff
@@ -49,6 +51,11 @@ public void onInit(SXRContext sxrContext) {
 
         SXRNode root = new SXRNode(sxrContext);
         SXRNode character = createCharacter(sxrContext);
+        character.getTransform().setScale(0.008f,0.008f,0.008f);
```
Comment (liaxim) [inline sxr-multilight/app/src/main/java/com/samsungxr/multilight/MultiLightMain.java]: Why are now the scale and the translation necessary?

File: sxr-arpet/app/build.gradle (hunk 2 above)
Comment (spodila1) [reply sxr-arpet/app/build.gradle]: @liaxim It did not work on my end until I changed the version. Now I have set it back.

File: common.gradle (hunk 1 above)
Comment (spodila1) [reply common.gradle]: @liaxim This change is not required. I have reverted

File: sxr-multilight/app/src/main/java/com/samsungxr/multilight/MultiLightMain.java (hunk 3 above)
Comment (spodila1) [reply sxr-multilight/app/src/main/java/com/samsungxr/multilight/MultiLightMain.java]: @liaxim Could you test it now. It is working.
